### ISSUE TITLE
Wrong Average Recall metric logged in MLFlowSink

### ISSUE THREAD

**Issue by tillfri** [roboflow/rf-detr#734]
### Search before asking

- [x] I have searched the RF-DETR issues and found no similar bug report.

### Bug

First of all, thanks for the continuous work on the repo, greatly appreciated :)

while the new integration of the MLFlow logging support works great, i think a small number/index missmatch has slipped in. 
in rfdetr/util/metrics.py in lines 377 and 389 it uses index "6", while in the other logging sinks it uses "8".
This leads to logging AR@maxDets=1 instead of AR@maxDets=100.

### Environment

- RF-DETR 1.5.0
- MacOS 
- Python 3.10
- PyTorch 2.8.0
- Apple M4

### Minimal Reproducible Example

```python
from rfdetr import RFDETRSegMedium

if __name__ == "__main__":
    model = RFDETRSegMedium()

    model.train(
        dataset_dir="...",
        output_dir="...",
        epochs=10,
        batch_size=1,
        grad_accum_steps=16, 
        resolution=432,
        device="mps",
        mlflow=True,
    )
```

### Are you willing to submit a PR?

- [x] Yes, I'd like to help by submitting a PR!
